addons/sitting/functions/fnc_canSit.sqf

```diff
@@ -20,6 +20,6 @@ params ["_seat", "_player"];
 
 // Sitting enabled, is seat object, not occupied and standing up (or not on a big slope)
 GVAR(enable) &&
-{isNil {_player getVariable QGVAR(isSitting)}} &&
-{!(_seat call ACEFUNC(common,owned))} &&
+{isNil {_player getVariable QGVAR(sittingStatus)}} &&
+{(_seat getVariable [QGVAR(seatsClaimed), []]) find true == -1} &&
```

Comment:
```suggestion
{
    private _seatsClaimed = _seat getVariable [QGVAR(seatsClaimed), []];
    _seatsClaimed isEqualTo []
    || {_seatsClaimed find false > -1}
} &&
```
I didn't test your code neither mine (will try later this day), but looks like your condition is `false` when any seat is claimed, so only one person would be able to multi-seat.

Reply:
Good point, I should just be checking that specific index (seat pos) or empty!